js/styles.js

```diff
@@ -29,14 +29,22 @@ function emphasizeCounty(selection) {
     .classed("emphasize-county", true);
 }
 
-function foregroundState(selection, scale = 1) {
+function foregroundState(selection, scale) {
```

Comment:
so were these the trick to #151? Setting a default to the functions?

Reply:
Seems like it